packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -41,12 +43,16 @@ export interface GraphNGProps extends Themeable2 {
   timeZone: TimeZone;
   legend: VizLegendOptions;
   fields?: XYFieldMatchers; // default will assume timeseries data
+  renderers?: Renderers;
+  tweakScale?: (opts: ScaleProps) => ScaleProps;
+  tweakAxis?: (opts: AxisProps) => AxisProps;
   onLegendClick?: (event: GraphNGLegendEvent) => void;
   children?: (builder: UPlotConfigBuilder, alignedFrame: DataFrame) => React.ReactNode;
   prepConfig: (alignedFrame: DataFrame, allFrames: DataFrame[], getTimeRange: () => TimeRange) => UPlotConfigBuilder;
   propsToDiff?: Array<string | PropDiffFn>;
   preparePlotFrame?: (frames: DataFrame[], dimFields: XYFieldMatchers) => DataFrame;
   renderLegend: (config: UPlotConfigBuilder) => React.ReactElement | null;
+  options?: Record<string, any>;
```

Comment:
Do i understand right that this is not actually used for anything -- only as trigger to `propsToDiff`?  If that is the case, lets make a big comment and maybe @dprokop has a better suggestion

Reply:
yes, this is correct.